packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/SendCryptoInput/SendCryptoSelect/index.tsx

```diff
@@ -71,7 +73,7 @@ const SendCryptoSelect = () => {
                                     }.svg`}
                                 />
                             )}
-                            <Label>{option.label}</Label>
+                            <Label>{areSatsUsed ? 'sat' : option.label}</Label>
```

Comment:
I guess since the test networks don't have exchange rates the network symbol can be excluded? Or better have it? @hynek-jina 

```suggestion
                            <Label>{areSatsUsed ? isTestnet ? `sat ${symbol?.toUpperCase()}` : 'sat' : option.label}</Label>
```

Reply:
I believe it's ok not to handle testnets here.

Reply:
I used the same code as in the Send form.